<launch>
    <param name="/use_sim_time" value="true"/>
    <node pkg="robot_navigation" type="sql_client" name="sql_client" output="screen" />
        
</launch>